app/serializers/user_serializer.rb

```diff
@@ -43,10 +23,6 @@ def max_subjects
     @model.subject_limit
   end
 
-  def uploaded_subjects_count
```

Comment:
i'll move to a query based count on each create, i'll index that field and make sure it's unscoped.